card/db/migrate_core_cards/20150116220620_add_markdown_type.rb

```diff
@@ -0,0 +1,7 @@
+# -*- encoding : utf-8 -*-
+
+class AddMarkdownType < Card::Migration
+  def up
+    ensure_card :name=>'Markdown', :codename=>'markdown', :type_id=>Card::CardtypeID
```

Comment:
do we autogenerate this old-school hash notation?

Reply:
no, it's original old-school from 2015 (see the timestamp in the migration's file name)